app/services/GoogleCalendarService.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+use Google\Client;
+use Google\Service\Calendar as GoogleCalendarService;
+use Google\Service\Calendar\Event as GoogleCalendarEvent;
+class GoogleCalendar
```

Comment:
Would be nice to name this GoogleCalendarService for clarity

Reply:
Right, just did that